tests/benchmarks/test_array.py

```diff
@@ -37,10 +37,10 @@ def test_anom_mean(small_client, new_array):
         dims=["time", "x"],
         coords={"day": ("time", np.arange(data.shape[0]) % ngroups)},
     )
-
-    clim = arr.groupby("day").mean(dim="time")
-    anom = arr.groupby("day") - clim
-    anom_mean = anom.mean(dim="time")
+    with xarray.set_options(use_flox=False):
```

Comment:
I'd suggest moving the xarray-related tests to `test_xarray.py`.

Reply:
I'd rather do this as a follow up, but the general idea is good